src/coreclr/jit/emitriscv64.cpp

```diff
@@ -588,7 +588,7 @@ void emitter::emitIns_R_R(
 {
     code_t code = emitInsCode(ins);
 
-    if (INS_mov == ins)
+    if (INS_mov == ins || INS_sext_w == ins)
```

Comment:
Have you discussed in your team to use pseudo code actively in JIT?
Actually, I want to remove all pseudo codes like `mov` in JITC because some pseudo codes (contains multiple normal instructions) cannot be used in JIT.

Reply:
IMO simple "renaming" pseudos like sext.w, bnez, mov, etc, are fine. We're already using them, they convey the intention a bit more clearly, and they make the assembly look closer to print-outs from native compilers.

I agree about multi-instruction pseudos, though.

Reply:
Okay. Actually, we used few pesudos in few code locations, however it is not printed properly in dump. Could you add such instructions like 'mov' and 'nop' in `emitDispInsName` too?
Thank you.

Reply:
I can add pseudos to the disassm but I would prefer to do it in other PR. Would you mind?